src/everest/detached/jobs/everserver.py

```diff
@@ -305,7 +304,8 @@ def main():
             simulation_callback=partial(_sim_monitor, shared_data=shared_data),
             optimization_callback=partial(_opt_monitor, shared_data=shared_data),
         )
-        if run_model._queue_config.queue_system == QueueSystem.LOCAL:
+
+        if config.simulator.queue_system.name == "local":
```

Comment:
Not sure this will fail, but:
```python
if run_model._queue_config.queue_system.lower() == QueueSystem.LOCAL.lower():
```
might be safer. Would we be able to test this?

Reply:
Why is it lower in the first place? 

I think we should merge this as it is as we know that will fix the problem at hand

Reply:
Because everest user lower while ert uses upper. Changed it for ert here: https://github.com/equinor/ert/commit/68ab17b46c095bb8fe8d14df0e147508e10f34b4, so should always be lower now.